src/qt/pivx/coldstakingwidget.cpp

```diff
@@ -136,8 +136,8 @@ ColdStakingWidget::ColdStakingWidget(PIVXGUI* parent) :
     ui->btnCoinControl->setTitleClassAndText("btn-title-grey", "Coin Control");
     ui->btnCoinControl->setSubTitleClassAndText("text-subtitle", "Select PIV outputs to delegate.");
 
-    ui->btnColdStaking->setTitleClassAndText("btn-title-grey", "Create Cold Stake Address");
-    ui->btnColdStaking->setSubTitleClassAndText("text-subtitle", "Creates an address to receive coin\ndelegations and be able to stake them.");
+    ui->btnColdStaking->setTitleClassAndText("btn-title-grey", "Create staking address");
```

Comment:
may be better to leave this mentioning that it is a cold staking address, so as to not introduce confusion.

Also, would prefer to keep window/dialog titles with each word's first character capitalized (proper name formatting)